spec/features/card_spec (copy).rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+require "spec_helper"
+require "capybara/rspec"
+
+describe "Card", type: "feature" do
+  let!(:user) { create(:user) }
+
+  before(:each) do
+   # login("user@email.com", "password")
+   login_user(user = nil)
```

Comment:
Useless assignment to variable - `user`.<br>Use 2 (not 1) spaces for indentation.